src/pineko/theory.py

```diff
@@ -388,6 +388,7 @@ def fk(self, name, grid_path, tcard, pdf):
             grid.lumi(),
         ):
             max_as += 1
+        # NB: This would not happen for simFONLL
```

Comment:
Should we also use *numerical FONLL* everywhere, to avoid the double nomenclature?

Reply:
Yes I am changing all the simFONLL to num_fonll now